src/Components/Components/ref/Microsoft.AspNetCore.Components.netstandard2.0.Manual.cs

```diff
@@ -44,4 +44,146 @@ namespace Microsoft.AspNetCore.Components.RenderTree
         public Microsoft.AspNetCore.Components.IComponent Component { get { throw null; } }
         public override string ToString() { throw null; }
     }
+}
+
+// Built-in components: https://github.com/aspnet/AspNetCore/issues/8825
+namespace Microsoft.AspNetCore.Components
+{
+    public partial class CascadingValue<T> : Microsoft.AspNetCore.Components.IComponent
+    {
+        public CascadingValue() { }
+        [Microsoft.AspNetCore.Components.ParameterAttribute]
+        public Microsoft.AspNetCore.Components.RenderFragment ChildContent { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } private set { throw null; }}
```

Comment:
adding the `private set { throw null; }` is what requires us to do these manually. Tooling and runtime ignore visibility and requires a setter.